react/src/components/BatchSessionScheduledTimeSetting.tsx

```diff
@@ -46,7 +46,7 @@ const BatchSessionScheduledTimeSetting: React.FC<Props> = ({
           <DatePickerISO
             {...datePickerISOProps}
             popupStyle={{ position: 'fixed' }}
-            disabledDate={(date) => {
+            disabledDate={(date: dayjs.Dayjs) => {
```

Comment:
please revet this line. :) @agatha197 
```suggestion
            disabledDate={(date) => {
```